### PR TITLE
BrickProfile: added APIs for dApp

### PR DESCRIPTION
**Pull request by Leechael**

### CODE REVIEW

File: phat/contracts/action_offchain_rollup/lib.rs [hunk 1]
```diff
@@ -282,6 +292,39 @@ mod action_offchain_rollup {
...
+        /// @category Configuration
+        ///
+        #[ink(message)]
+        pub fn get_configuration(&self) -> Configuration {
+            let mut _rpc = None;
+            let mut _client_addr = None;
+            if self.client.is_some() {
+                _rpc = Some(self.client.as_ref().unwrap().rpc.clone());
+                _client_addr = Some(self.client.as_ref().unwrap().client_addr);
+            }
```
Comment (kvinwang) [inline phat/contracts/action_offchain_rollup/lib.rs:307]: Lets avoid the unwrap:
```suggestion
            let rpc = self.client.as_ref().map(|c| c.rpc.clone());
            let client_addr = self.client.as_ref().map(|c| c.rpc.client_addr);
```

File: phat/contracts/brick_profile/lib.rs [hunk 2]
```diff
@@ -166,14 +217,62 @@ mod brick_profile {
             Ok(id)
         }
 
+        /// Adds a new workflow and authorizes it to use the account, only owner is allowed
+        ///
+        /// @category Workflow
+        #[ink(message)]
+        pub fn add_workflow_and_authorized(&mut self, name: String, commandline: String, account: ExternalAccountId) -> Result<WorkflowId> {
```
Comment (h4x3rotab) [inline phat/contracts/brick_profile/lib.rs:224]:
```suggestion
        pub fn add_workflow_and_authorize(&mut self, name: String, commandline: String, account: ExternalAccountId) -> Result<WorkflowId> {
```

File: phat/contracts/brick_profile/lib.rs [hunk 3]
```diff
@@ -200,14 +302,43 @@ mod brick_profile {
...
             let sk = pink_web3::keys::pink::KeyPair::from(account.sk);
             Ok(sk.address())
         }
 
+        /// Get all EVM account addresses, aonly owner is allowed
+        ///
+        /// @category EvmAccount
+        ///
+        #[ink(message)]
+        pub fn get_all_evm_account(&self) -> Result<Vec<ExternalAccountInfo>> {
```
Comment (h4x3rotab) [inline phat/contracts/brick_profile/lib.rs:320]:
```suggestion
        pub fn get_all_evm_accounts(&self) -> Result<Vec<ExternalAccountInfo>> {
```

**Review by h4x3rotab — APPROVED** [review]
LGTM

**Pull request merged**